test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -249,21 +243,37 @@ def api_client
 
       # Here we have obtained the creds, but for the mock, we will leave the
       # channel insecure.
-      @grpc_stub.new(@mock_host, :this_channel_is_insecure)
+      @grpc_stub
     end
   end
 
   # GRPC logging mock that successfully logs the records.
-  class GRPCLoggingMockService < Google::Logging::V2::LoggingServiceV2::Service
+  class GRPCLoggingMockService <
+      Google::Cloud::Logging::V2::LoggingServiceV2Client
     def initialize(requests_received)
       super()
       @requests_received = requests_received
     end
 
-    def write_log_entries(request, _call)
+    # rubocop:disable Lint/UnusedMethodArgument
+    # rubocop:disable Metrics/ParameterLists
+    def write_log_entries(entries,
+                          log_name: nil,
+                          resource: nil,
+                          labels: nil,
+                          partial_success: nil,
+                          options: nil)
```

Comment:
If `options` is unused, do we even need to specify it?

Reply:
Yeah, the mock needs to match exactly the original method arguments.